syntax = "proto2";
option optimize_for = LITE_RUNTIME;
option java_package = "ProtoMessages";
option java_outer_classname = "MessageRequestPingBase";

message MessageRequestPing {
required string someText = 1 [default = ""];
}